zeppelin-plugins/notebookrepo/filesystem/pom.xml

```diff
@@ -43,6 +43,10 @@
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-client</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Based on the Hadoop profile in zeppelin-zengine, the Hadoop lib may or may not be present.
We should also put Hadoop profiles in this `pom.xml` and explicitly set this dependency to `provided`.

Reply:
still valid